src/run/next.cts

```diff
@@ -97,11 +92,9 @@ export async function getMockedRequestHandler(...args: Parameters<typeof getRequ
       } catch (error) {
         // only try to get .html files from the blob store
         if (typeof path === 'string' && path.endsWith('.html')) {
-          const store = getRegionalBlobStore()
+          const cacheStore = getMemoizedKeyValueStoreBackedByRegionalBlobStore()
```

Comment:
Thoughts on using `getStore<HtmlBlob>.get` instead of `getStore.get<HtmlBlob>` (not putting the full name)?